lib/MusicBrainz/Server/Data/Statistics.pm

```diff
@@ -775,13 +775,17 @@ my %stats = (
         CALC => sub {
             my ($self, $sql) = @_;
 
-            my $data = $sql->select_list_of_lists(
-                q{SELECT COALESCE(l.iso_code_3::text, 'null'), COUNT(wl.work) AS count
-                FROM work_language wl FULL OUTER JOIN language l
-                    ON wl.language=l.id
-                WHERE l.iso_code_2t IS NOT NULL OR l.frequency > 0
-                GROUP BY l.iso_code_3},
-            );
+            my $data = $sql->select_list_of_lists(<<~'SQL');
+                         SELECT COALESCE(l.iso_code_3::text, 'null'),
+                                COUNT(w.gid) AS count
+                           FROM work w
+                      LEFT JOIN work_language wl ON wl.work=w.id
+                FULL OUTER JOIN language l ON wl.language=l.id
+                          WHERE l.iso_code_2t IS NOT NULL
+                             OR l.frequency > 0
+                             OR l.id IS NULL
```

Comment:
Same comment as for the the first commit: It seems worth splitting the large coding style changes off from this third commit <https://github.com/metabrainz/musicbrainz-server/pull/2572/commits/ee31d890210621a95a89021e265986839f018cee> to make more visible this only change actually related to the commit message.

Reply:
This is not the only change here, but that makes it obvious that mixing the two is confusing :) Splitting.